ui/src/app/components/workflows/workflow/workflow.component.ts

```diff
@@ -159,13 +160,22 @@ export class WorkflowComponent implements OnInit, OnDestroy {
       this.confirmationDialogServiceSubscription = this.confirmationDialogService
         .confirm(ConfirmationDialogTypes.YesOrNo, texts.UPDATE_WORKFLOW_CONFIRMATION_TITLE, texts.UPDATE_WORKFLOW_CONFIRMATION_CONTENT)
         .subscribe((confirmed) => {
-          if (confirmed) this.store.dispatch(new UpdateWorkflow());
+          if (confirmed) {
+            this.store.dispatch(new UpdateWorkflow());
+            this.initializeAfterAction();
+          }
         });
     } else {
       this.showHiddenParts();
     }
   }
 
+  initializeAfterAction() {
+    setTimeout(() => {
+      this.store.dispatch(new InitializeWorkflows());
```

Comment:
This looks fishy to me :) Why do you need to invoke Initialize after Update? Can't you sort the projects in the `UPDATE_WORKFLOW_SUCCESS` case of the reducer?